apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -425,11 +427,31 @@ export class ApiModelGenerator {
         startingNode: astDeclaration.declaration,
         nodesToCapture
       });
-      const docComment: tsdoc.DocComment | undefined = this._collector.fetchMetadata(astDeclaration).tsdocComment;
-      const releaseTag: ReleaseTag = this._collector.fetchMetadata(astDeclaration.astSymbol).releaseTag;
-
-      apiFunction = new ApiFunction({ name, docComment, releaseTag, typeParameters, parameters, overloadIndex,
-        excerptTokens, returnTypeTokenRange });
+      const declarationMetadata: DeclarationMetadata = this._collector.fetchMetadata(astDeclaration);
+      const docComment: tsdoc.DocComment | undefined = declarationMetadata.tsdocComment;
+      const effectiveReleaseTag: ReleaseTag = declarationMetadata.effectiveReleaseTag;
+      if (effectiveReleaseTag === ReleaseTag.Internal || effectiveReleaseTag === ReleaseTag.Alpha) {
+        return; // trim out items marked as "@internal" or "@alpha"
+      }
+      // If we have lower-release function overloads, include that information
+      const symbolMetadata: SymbolMetadata = this._collector.fetchMetadata(astDeclaration.astSymbol);
+      const releaseTag: ReleaseTag = (
```

Comment:
Given that we now fallback to the parent release tag if one wasn't specified, I'm wondering if this check is even needed anymore?

Reply:
Right, we can remove it.